tests/inputs/GliderTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace inputs;
+
+use GameOfLife\Field;
+use GameOfLife\inputs\Glider;
+use GameOfLife\outputs\Console;
+use PHPUnit\Framework\TestCase;
+use Ulrichsg\Getopt;
+
+require_once "Getopt.php";
+
+class GliderTest extends TestCase
+{
+    public function testFillFieldYieldsNotEmptyField()
+    {
+        $field = new Field(5, 5, 1);
+        $glider = new Glider();
+        $options = new Getopt();
+        $console = new Console();
+
+        $glider->fillField($field, $console, $options);
+        $isEmpty = true;
+
+        for ($y = 0; $y < $field->height(); $y++)
+        {
+            for ($x = 0; $x < $field->width(); $x++)
+            {
+                if ($field->fieldValue($x, $y) != 0)
+                {
+                    $isEmpty = false;
+                }
+            }
+        }
+
+        $this->assertFalse($isEmpty);
+
+        return $field;
+    }
+
+    /**
+     * @depends testFillFieldYieldsNotEmptyField
+     * @param Field $field
+     */
+    public function testGliderPattern(Field $field)
+    {
+        $patter = [
```

Comment:
Correct name would be `$pattern`.
To help you with the unit-tests it would be nice if your Field class would have a static method `fromArray()`.
This method should construct a Field and initialize it with the data that is passed to it as parameter.
This would then allow you to write unit-test code like this:

```php
$expectedField = Field::createFromArray([
             [0, 0, 0, 0, 0],
             [0, 0, 1, 0, 0],
             [0, 0, 0, 1, 0],
             [0, 1, 1, 1, 0],
             [0, 0, 0, 0, 0]]);

$this->assertTrue($field->isEqualTo($expectedField));
```
So please implement that static method inside Field and use it wherever you can inside unit-tests to compare expected fields.